nym-connect/src/types/connection.ts

```diff
@@ -1,8 +1,3 @@
-export enum ConnectionStatusKind {
-  disconnected = 'disconnected',
-  disconnecting = 'disconnecting',
-  connected = 'connected',
-  connecting = 'connecting',
-}
+export type ConnectionStatusKind = 'disconnected' | 'disconnecting' | 'connected' | 'connecting';
```

Comment:
Like I said above, I'm less keen on these, because they are harder to search and replace in code. Mostly just a personal preference, there isn't a difference in runtime behaviour.